### ISSUE TITLE
Production data 

### ISSUE THREAD

**Issue by MarkMusumba** [weecology/retriever#1168]
I am requesting help in securing production data from FAO . 
http://www.fao.org/fishery/statistics/global-capture-production/en
MM

**Comment by ethanwhite** [comment]
Hi @MarkMusumba - thanks for getting in touch about this. It took a little bit because some of the files contain four different alphabets which causes some encoding challenges, but @henrykironde figured it out for most cases and I just merged his PR (#1169). This works for all of our back ends except for MySQL, so unless you needed that specific back end you should be able to install this now by running:

```
retriever update
retriever install sqlite fao-global-capture-product
```

substituting whatever output format you want for `sqlite`.

Let us know if you run into any issues.

**Comment by henrykironde** [comment]
And for the case of CSV use `retriever install csv fao-global-capture-product`
